templates/takeovers/_compliance-webinar.html

```diff
@@ -0,0 +1,26 @@
+<section lang="en" data-lang="{{ locale }}" class="p-strip--image is-deep js-takeover is-dark p-takeover--compliance {% if not default %}u-hide{% endif %}"
+    {% if default %}data-default="true" {% endif %}>
+    <div class="row u-vertically-center">
+        <div class="col-6 u-fade-left--medium">
+            <h1 style="font-weight: 100">Securing the enterprise</h1>
+            <p class="u-no-padding--top  u-sv3">Learn how Ubuntu’s trusted security and compliance solutions are at the forefront of solving regulatory issues across a wide range of industries.</p>
+            <p class="u-hide--large u-hide--medium u-align--center"><img src="{{ ASSET_SERVER_URL }}2217d1c8-Security.svg" alt="Compliance Webinar" class="p-compliance-icon u-fade-up"></p>
+            <p><a href="https://www.brighttalk.com/webcast/6793/340102?utm_source=Takeover&utm_medium=Takeover&utm_campaign=2)FY19_Cloud_Openstack_WBN_Compliance
```

Comment:
Looks like a typo made its way into the brief, please remove the "2)" from utm_campaign, thanks!